edgemesh/pkg/proxy/proxy.go

```diff
@@ -290,9 +290,16 @@ func (at *addrTable) getAddrTable(key string) *serviceTable {
 // filterResourceType implement filter. Proxy cares "Service" and "ServiceList" type
 func filterResourceType(msg model.Message) []v1.Service {
 	svcs := make([]v1.Service, 0)
-	content, ok := msg.Content.([]byte)
-	if !ok {
-		return svcs
+	var content []byte
+	var err error
+	switch msg.Content.(type) {
+	case []byte:
+		content = msg.GetContent().([]byte)
+	default:
+		content, err = json.Marshal(msg.GetContent())
+		if err != nil {
+			klog.Errorf("marshal message to edgemesh failed, err: %v", err)
+		}
```

Comment:
Will cause any bugs without this change? I've seen this code block(L293-302) many times...  ;-)

Reply:
Will there be different types if context that send to edgemesh？

Reply:
![image](https://user-images.githubusercontent.com/28776356/69958352-9418d600-153f-11ea-87ef-09f4280e7974.png)

Reply:
![image](https://user-images.githubusercontent.com/28776356/69958393-ad218700-153f-11ea-823b-b79fdb9949dc.png)